sdk/internal/testframework/request_matcher.go

```diff
@@ -0,0 +1,176 @@
+// +build go1.13
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package testframework
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"reflect"
+
+	"github.com/dnaeon/go-vcr/cassette"
+)
+
+type RequestMatcher struct {
+	context        TestContext
+	IgnoredHeaders map[string]*string
```

Comment:
Does the value `*string` have actual usage, or is the idea to use this as a hash set?  If it's the latter, the convention is to use `struct{}`.

Reply:
Thanks - I was wondering what the convention was.